.eslintrc.json

```diff
@@ -8,7 +8,7 @@
   },
   "extends": "eslint:recommended",
   "rules": {
-    "indent": ["error", 2],
+    "indent": ["off"],
```

Comment:
No need to do linting on indent, because prettier enforces its own rules. (The rules are different between eslint and prettier when looking at switch/case statements)